response_operations_ui/views/messages.py

```diff
@@ -195,6 +213,28 @@ def view_selected_survey(selected_survey):
                                response_error=True)
 
 
+@messages_bp.route('/threads/<thread_id>/close-conversation', methods=['GET', 'POST'])
```

Comment:
I m not sure I agree with the use of any kind of verb in a uri . the resource should resource based an dnot action based. Please check this nice example : https://stackoverflow.com/questions/18233632/rest-uri-design-for-changing-the-status-for-resource

Reply:
So, this route brings up the close-conversation speedbump page which lets the user confirm that they want to close and it contains some message data (ref, id etc) so not sure what this could be changed to

Reply:
ok just realised it already wrongly set up like this elsewhere. Let s keep consistent and leave it like it is then.